src/styles/main.scss

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+}
+
+.conteiner {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+}
+
+.watch {
+  position: relative;
+  height: 350px;
+  width: 350px;
+  background-color: aquamarine;
+  background-image: url(./gerb.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+  border-radius: 50%;
+  &__second {
+    height: 150px;
+    width: 5px;
+    background-color: blue;
+    border-radius: 2px;
+    position: absolute;
+    left: calc(50% - 2.5px);
+    top: calc(50% - 150px);
+    transform-origin: bottom;
+    animation: watch-hand 60s linear infinite;
+  }
+  &__minute {
+    height: 120px;
```

Comment:
Check 9th point in the [checklist](https://github.com/mate-academy/layout_stop-watch/blob/master/checklist.md)